main.go

```diff
@@ -136,7 +141,12 @@ func main() {
 				CaAddress:      common.HexToAddress(caAddress),
 			}
 
-			rpcServer, _, _, err := rpc.InitChainServiceAndRunRpcServer(pkString, chainOpts, useDurableStore, durableStoreFolder, useNats, msgPort, rpcPort)
+			ss, err := fs.Sub(fs.FS(staticSite), "packages/nitro-gui/dist")
```

Comment:
Is it important that we only do this in `main`? Or could we call `fs.Sub` right in the websocket server, avoiding passing `ss` around.

Reply:
The `main` entrypoint is where the files are embedded. The `.go` file with the `embed` directive (L19( cannot search up the directory tree, so without rearranging the repo we can't `embed` in (say) the `rpc` package or websocket server. That's the thing that necessitates our passing _something_ down to the rpc server from this file. 

Given that we have to embed or "import" the static site in `main.go`, we should do whatever wrangling we need to prepare it to be passed to the rpc server in `main.go`, too. That's what this line is doing, essentially trimming down the unnecessary path prefix.



I've tidied this up a bit [f2c7ed8](https://github.com/statechannels/go-nitro/pull/1449/commits/f2c7ed87b903d84a112241b13258e5860a415f33)